e2eAIOK/deltatuner/deltatuner/tuner/deltatuner_ssf.py

```diff
@@ -80,6 +80,48 @@ class SSFConfig(PeftConfig):
     def __post_init__(self):
         self.peft_type = DeltaTunerType.SSF
 
+    @classmethod
+    def from_pretrained(cls, pretrained_model_name_or_path: str, subfolder: Optional[str] = None, **kwargs):
```

Comment:
Can I ask where we will call this function, I didn't find it in deltatuner_model

Reply:
Yes, it's in the `DeltaTunerModel.from_pretrained` function, please check the following code:
```python
if peft_type_name in ['SSF']:
    config = DELTATUNER_TYPE_TO_CONFIG_MAPPING[peft_type_name].from_pretrained(model_id, **kwargs)
```